pkg/kubelet/volumemanager/populator/desired_state_of_world_populator_test.go

```diff
@@ -47,6 +47,122 @@ import (
 	"k8s.io/kubernetes/pkg/volume/util/types"
 )
 
+func TestFindAndAddNewPods_WithReprocessPodAndVolumeRetrievalError(t *testing.T) {
```

Comment:
Could you add another test showing that if pvc/pv could not be retrieved at first time, podPreviouslyProcessed and podHasBeenSeenOnce should return false.

Reply:
Please take a look at TestFindAndAddNewPods_WithVolumeRetrievalError